tests/models/test_arch_sccnn.py

```diff
@@ -0,0 +1,41 @@
+"""Unit test package for SCCNN."""
+import numpy as np
+import torch
+
+from tiatoolbox import utils
+from tiatoolbox.models.architecture import fetch_pretrained_weights
+from tiatoolbox.models.architecture.sccnn import SCCNN
+from tiatoolbox.wsicore.wsireader import WSIReader
+
+
+def _load_sccnn(tmp_path, name):
+    model = SCCNN()
+    fetch_pretrained_weights(name, f"{tmp_path}/weights.pth")
+    map_location = utils.misc.select_device(utils.env_detection.has_gpu())
+    pretrained = torch.load(f"{tmp_path}/weights.pth", map_location=map_location)
+    model.load_state_dict(pretrained)
+
+    return model
+
+
+def test_functionality(remote_sample, tmp_path):
+    """Functionality test."""
```

Comment:
Could you maybe describe what this means a bit on the next line? I assume you mean take in and image and perform inference then check the output is sensible?

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/434/commits/906d7516cc04d63cdbd8831be00961bccef73318